invenio_imoox/cli.py

```diff
@@ -21,7 +21,7 @@
 
 def get_records_from_imoox(endpoint: str) -> dict:
     """Get the response from imoox."""
-    response = requests.get(endpoint)
+    response = requests.get(endpoint, timeout=10)
```

Comment:
Since we're cleaning up this function:
- could add `response.raise_for_status()` to fail early when the response's HTTP-status-code signifies an error
- should probably use `requests`'s built-in `response.json()` in the next line